bindings/azure/eventhubs/eventhubs.go

```diff
@@ -81,14 +82,9 @@ func (a *AzureEventHubs) Invoke(ctx context.Context, req *bindings.InvokeRequest
 func (a *AzureEventHubs) Read(ctx context.Context, handler bindings.Handler) error {
 	// Start the subscription
 	// This is non-blocking
-	return a.AzureEventHubs.Subscribe(ctx, a.AzureEventHubs.EventHubName(), false, func(ctx context.Context, data []byte, metadata map[string]string) error {
-		res := bindings.ReadResponse{
-			Data:     data,
-			Metadata: metadata,
-		}
-		_, hErr := handler(ctx, &res)
-		return hErr
-	})
+	topic := a.AzureEventHubs.EventHubName()
+	bindingsHandler := a.AzureEventHubs.GetBindingsHandlerFunc(topic, false, handler, 1*time.Minute)
+	return a.AzureEventHubs.Subscribe(ctx, topic, 1, impl.DefaultMaxBulkSubAwaitDurationMs, bindingsHandler)
```

Comment:
Why `maxBulkSubCount` is hard-coded as 1 here?
If there is a logic behind it, can we put some relevant comment for it at an appropriate place?

Reply:
Since bindings doesn't have bulk handler, that's why maxBulkSubCount is hard-coded to 1 here. Similar to azure service bus: https://github.com/dapr/components-contrib/blob/31ccb5f1690ffb3435168e5233ae5e0511068376/bindings/azure/servicebusqueues/servicebusqueues.go#L107C12-L107C12

Reply:
Adding the relevant comment for it, thanks!